org.adempiere.base/src/org/compiere/process/ProcessInfo.java

```diff
@@ -691,6 +692,80 @@ public ProcessInfoLog[] getLogs()
 		return logs;
 	}	//	getLogs
 
+	
+	/**
+	 * Save Log to DB
+	 * @param Log_ID
+	 * @param P_ID
+	 * @param P_Date
+	 * @param P_Number
+	 * @param P_Msg
+	 * @param tableId
+	 * @param recordId
+	 */
+	public void saveLog (int Log_ID, int P_ID, Timestamp P_Date, BigDecimal P_Number, String P_Msg,int tableId,int recordId)
+	{
+		saveLog (new ProcessInfoLog (Log_ID, P_ID, P_Date, P_Number, P_Msg,tableId,recordId));
+	}
+	
+	/**
+	 * Save Log to DB
+	 * @param P_ID
+	 * @param P_Date
+	 * @param P_Number
+	 * @param P_Msg
+	 * @param tableId
+	 * @param recordId
+	 */
+	public void saveLog (int P_ID, Timestamp P_Date, BigDecimal P_Number, String P_Msg ,int tableId,int recordId)
+	{
+		saveLog (new ProcessInfoLog (P_ID, P_Date, P_Number, P_Msg,tableId, recordId));
+	}
+	
+	/**************************************************************************

```

Comment:
Just curious, why different style of javadoc is being used in the same file ?

Reply:
That is a copy-paste issue...